assets/javascripts/wizard/initializers/register-widgets.js.es6

```diff
@@ -0,0 +1,276 @@
+import { escapeExpression, formatUsername } from "discourse/lib/utilities";
```

Comment:
@fzngagan A few issues here. Firstly, I want to be sure there's a good reason for us to be importing a new framework (i.e. the widget framework) into the custom wizard app. How does it help us here? Secondly, there's no need to recreate our own version of discourse widgets in the custom wizard app if we're already importing the widget system, we may as well just import the original version and that override them where needed if we're going that route. Let's discuss this.

Reply:
just try to replicate the Ui with handlebars